src/bindings/python/README.md

```diff
@@ -0,0 +1,48 @@
+# OpenVINO Python API
+
+OpenVINO Python API is a part of OpenVINO library. The component is responsible for:
+
+* Bindings of OpenVINO - allowing users to utilize OpenVINO library in their Python code. Python API provides bindings to basic and advanced APIs from OpenVINO Runtime.
+* Extending OpenVINO with pythonic features - on top of direct translations from C++, Python API component:
+    * Adds specific extensions to support numpy-based data.
+    * Provides support for external frameworks inputs.
+    * Provides shortcuts and helpers with more pythonic design.
+    * Allows to apply advanced concepts like shared memory to take full advantage of OpenVINO.
+
+OpenVINO Python API uses [common codestyle checks](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/docs/contributing.md#contribution-guidelines-and-best-practices) that are adjusted to project needs.
+
+## Key contacts
+
+If you have any questions, feature requests or want us to review your PRs, send us a message or ping us on GitHub via [openvino-ie-python-api-maintainers](https://github.com/orgs/openvinotoolkit/teams/openvino-ie-python-api-maintainers). You can always directly contact everyone from this group.
```

Comment:
nitpick: If I remember right you cannot write a message on github group :)

Reply:
Most of us have corporate emails shown in their profiles, thus I pointed to direct contact as an option.